### ISSUE TITLE
low validation accuracy with pre-train model on kinetic dataset

### ISSUE THREAD

**Issue by jeremy43** [kenshohara/3D-ResNets-PyTorch#43]
Hi，when I try to apply the pre-train model(resnext-101-64f-kinetics.pth)on the validation set from Kinetic dataset, the accuracy turns out to be very low(much like random prediction). I have check the loader and did not modify the code.  I am wondering  how does the pre-train model come from.  Did you directly train Pytorch code from zero or transfer the weights from Torch-version pre-train model?  Thx!

**Comment by kenshohara** [comment]
I trained the model from scratch using the pytorch codes,
and I confirmed my pretrained models achieved the accuracies reported in my paper.
Did you correctly specify using the pretrained models?

**Comment by zy486at189cn** [comment]
How many days do you compute at one GTX titan X?@kenshohara

**Comment by kenshohara** [comment]
It took 3 weeks using 8 GPUs.

**Comment by kenshohara** [comment]
@jeremy43 
Issue #45 may be the same issue as yours.

**Comment by jeremy43** [comment]
Hi， I choose the resume_path instead, the problem is solved, thanks!

**Comment by Fazlik995** [comment]
@jeremy43 
Did you achieve the accuracy reported in the paper? (94.5%)

If so, could you tell how?

My clip-level accuracy (val.log) is about 86%, however, video-level accuracy is about 88%.

I cannot understand why?

I use to evaluation the below code:
python main.py --root_path ~/project/ucf101 --video_path jpg0 --annotation_path ucf101_01.json
--result_path results/333 --dataset ucf101 --n_classes 101
--model resnext --model_depth 101 --resnet_shortcut B --resnext_cardinality 32 --sample_duration 16
--resume_path /results/save_200.pth --batch_size 64 --n_threads 12 --no_train --no_val --test --test_subset val

Thank you
